pyquil/operator_estimation.py

```diff
@@ -768,27 +798,48 @@ def measure_observables(qc: QuantumComputer, tomo_experiment: TomographyExperime
         for qubit, op_str in max_weight_out_op:
             total_prog += _local_pauli_eig_meas(op_str, qubit)
 
-        # 2. Run the experiment
-        bitstrings = qc.run_and_measure(total_prog, n_shots)
+        if symmetrize:
```

Comment:
point still stands about refactoring/merging with `QuantumComputer.run_symmetrized_readout` to avoid duplicated code

Reply:
Point noted. However, it's not immediately obvious (to me, anyway) how to fit `QuantumComputer.run_symmetrized_readout` *as it currently exists* into exhaustive symmetrization, i.e. run 2^n instead of 2*n Programs involving measurements of n qubits. We can modify that function and then call it inside measure_observables(..), but I'm not entirely sure if that's safe or if it might inadvertently break something else. Perhaps this extends outside of the scope of this PR, and can be addressed when we promote `measure_observables` (or some close cousin) to a method of `QuantumComputer`? I can make an issue to address it in the future if that makes sense.